mjml/elements/mj_column.py

```diff
@@ -205,12 +205,13 @@ def render_child(component):
             if component.isRawElement():
                 return component.render()
             td_attrs = component.html_attrs(
-                align = component.getAttribute('align'),
+                align = component.getAttribute('align', True),
                 # vertical-align
-                class_ = component.getAttribute('css-class'),
+                class_ = component.getAttribute('css-class', True),
```

Comment:
Why do you use `True` as default here? That should be a css class name? Similar issue with "background". You used a default value `True` for `.getAttribute(...)` in some places but I'm not sure why these were necessary?
I'm sorry for noticing that so late and maybe it's just my end-of-year brain shutdown. This is really the last issue I'm going to raise about this PR.

Reply:
`True` isn't the default value, it's setting the `missing_ok` argument (otherwise an error is thrown when attempting to retrieve css-class, since not every component supports it). I could probably specify `missing_ok=True` instead for clarity?

Reply:
Yes, please. Seems like my brain is hard-wired to expect that  the second parameter of `.get...()` is the default value so that would help. Maybe make `missing_ok` a keyword-only parameter in the function signature.